packages/@storefront/sayt/src/previous-search-terms/index.html

```diff
@@ -0,0 +1,3 @@
+<gb-list items="{state.previousSearches}" item-alias="previous-search" item-props="{ onClick: props.onClick }">
+  <yield/>
```

Comment:
This should probably have some kind of default output (ie. something *within* the `<yield />` tag).